tools/tasks.py

```diff
@@ -24,6 +24,19 @@
 import urllib.request
 
 
+# The 'latin-1' encoding to map byte values directly to the first 256 Unicode code points.
+# This is the closest equivalent Python 3 offers to the permissive Python 2 text handling model.
+# While the Windows cp1252 encoding is also sometimes referred to as 'latin-1', it doesn't map
+# all possible byte values, and thus needs to be used in combination with the 'surrogateescape'
+# error handler to ensure it never throws UnicodeDecodeError.
```

Comment:
I'm not sure we need these paragraphs lifted straight from the Python docs. We can always look them up if nessesary. If it's valuable information, opt to include a URL instead:

http://python-notes.curiousefficiency.org/en/latest/python3/text_file_processing.html#files-in-an-ascii-compatible-encoding-best-effort-is-acceptable

If anything, we should include a comment as to _why_ we're preferring `latin-1` over `utf-8`, like the comment in cbcollect_info:

https://github.com/couchbase/ns_server/blob/c294e997b6a192a1f0067ec4e979c59c156632f9/cbcollect_info#L94-L104

Reply:
Makes sense, refactored comments.